src/pkrd/rng/mt.rs

```diff
@@ -0,0 +1,77 @@
+pub struct MT {
```

Comment:
I think it would be good to add a smoke test to the bottom of the file, just to make sure future updates don't break this in unexpected ways:
```rust
#[cfg(test)]
mod test {
    use super::*;
    use alloc::{vec, vec::Vec};

    #[test]
    fn should_advance_as_expected() {
        let rng = MT::new(0xaabbccdd);
        let result: Vec<u32> = rng.take(100).collect();

        let expected_result = vec![/* the expected result */];
        assert_eq!(result, expected_result);
    }
}
```

Reply:
I can add some sort of test